packages/unmock-core/src/console/index.ts

```diff
@@ -1,7 +1,2 @@
-import { CustomConsole } from "./console";
-
-const customConsole = new CustomConsole(process.stdout, process.stderr);
-
-export { CustomConsole, formatMsg } from "./console";
+export { formatMsg } from "./console";
 export { Chalks } from "./chalks";
```

Comment:
Is `Chalks` still used anywhere?

Reply:
Not sure but I think it's logical for the `console` module to export it.